src/test/java/io/github/borewit/sanitize/SVGSanitizerTest.java

```diff
@@ -77,6 +77,7 @@ void generateJsonHashMap() throws Exception {
             "form-action3.svg",
```

Comment:
There's still a test called `tmp()` for just this file, which appears to be similar to `sanitizeToInputStream()` (which also includes this file).